fixtures/bugs/1719/pkg/models/user.go

```diff
@@ -0,0 +1,46 @@
+// Code generated by go-swagger; DO NOT EDIT.
```

Comment:
how come this file is included?
wouldn't it be overwritten every test

Reply:
I  manually generated some sample models to use as "external packages" from the main fixture spec.
I may remove the "DO NOT EDIT" line if you found this misleading.

Reply:
I was just curious

Reply:
ok good